docs/csharp/tour-of-csharp/tutorials/branches-and-loops.yml

```diff
@@ -113,7 +113,7 @@ items:
     else
     {
         Console.WriteLine("The answer is not greater than 10");
-        Console.WriteLine("Or the first number is not equal to the second");
+        Console.WriteLine("And the first number is not equal to the second");
```

Comment:
The original statement is correct. In the `else` branch, it means that the `if` condition evaluates to false. When you have `A && B` and it evaluates to false, then either `A` or `B` or both evaluates to false. (This originates from De Morgan's law)

Reply:
Thank you for the feedback. You're right, the changes proposed will be revoked.